st2common/tests/unit/migrations/test_v35_migrate_db_dict_field_values.py

```diff
@@ -80,10 +80,20 @@ def test_migrate_executions(self):
 
         class ActionExecutionDB_OldFieldType(ActionExecutionDB):
             result = stormbase.EscapedDynamicField(default={})
+            liveaction = stormbase.EscapedDictField(required=True)
+            parameters = stormbase.EscapedDynamicField(default={})
 
         class LiveActionDB_OldFieldType(LiveActionDB):
             result = stormbase.EscapedDynamicField(default={})
 
+        #todo(aj) need ActionExecutionDB_NewFieldType to be used for update
```

Comment:
todo: need to hard code the state of this model so this unit test works again

Reply:
Is this test worth keeping around?  It assumes that the current model structure is the final model structure ever.